packages/amplify-util-mock/src/api/api.ts

```diff
@@ -409,38 +410,40 @@ export class APITest {
     } catch (error) {
       throw new AmplifyFault('MockProcessFault', {
         message: 'Unable to start the local OpenSearch Instance.',
+        details: error?.message || '',
         link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url
       });
     }
   }
 
   private async clearAllIndices(openSearchURL:URL) {
-    if (openSearchURL) {
-      const errMessage = 'Unable to Clear the local OpenSearch Indices.';
-      try {
-        const url = openSearchURL.toString() + '*';
-
-        const result = await fetch(url, {
-            method: 'DELETE',
-            headers: {
-                'Content-type': 'application/json',
-            }
-        });
-        const status = await result.json();
-        if (!(status?.acknowledged)) {
-          throw new AmplifyFault('MockProcessFault', {
-            message: errMessage,
-            link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url
-          });
-        }
-      }
-      catch(error) {
+    if (!openSearchURL) {
+      return;
+    }
+    const errMessage = 'Unable to Clear the local OpenSearch Indices.';
+    try {
+      const url = openSearchURL.toString() + '*';
+      const result = await fetch(url, {
+          method: 'DELETE',
+          headers: {
+              'Content-type': 'application/json',
+          }
+      });
+      const status = await result.json();
+      if (!(status?.acknowledged)) {
         throw new AmplifyFault('MockProcessFault', {
-          message: errMessage,
+          message: 'The action to delete all items in an index is not acknowledged by the Opensearch server.',
           link: AMPLIFY_SUPPORT_DOCS.CLI_GRAPHQL_TROUBLESHOOTING.url
         });
       }
     }
+    catch(error) {
+      throw new AmplifyFault('MockProcessFault', {
+        message: errMessage,
+        details: error?.message || '',
```

Comment:
this shouldn't be necessary if passing the error as the 3rd arg to the AmplifyFault constructor